packages/mobile/src/identity/reducer.ts

```diff
@@ -383,6 +383,8 @@ export const acceptedAttestationCodesSelector = (state: RootState) =>
   state.identity.acceptedAttestationCodes
 export const e164NumberToAddressSelector = (state: RootState) => state.identity.e164NumberToAddress
 export const addressToE164NumberSelector = (state: RootState) => state.identity.addressToE164Number
+export const addressToDataEncryptionKeySelector = (state: RootState) =>
+  state.identity.addressToDataEncryptionKey
```

Comment:
This looks unrelated to this PR, maybe a merge result of a bad merge conflict?

Reply:
I had to define this selector to use in `SendConfirmation`!

Reply:
Oh right, missed that because the GitHub UI collapsed that file 👀